main.qml

```diff
@@ -60,7 +60,12 @@ ApplicationWindow {
     property var currentWallet;
     property bool disconnected: currentWallet ? currentWallet.disconnected : false
     property var transaction;
+    property var transactionAmount;
```

Comment:
It should be possible to remove all of these including `transactionDescription`.